src/helperise/string_helpers/fi_ro_po.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @param args receives three strings parent string and the string to be replaced.
+ * @returns string
+ */
+function fiRopo (args) {
+    if (Array.isArray(args)) {
+        if (args.length === 3) {
```

Comment:
Hey bro, no need to check args.length.
If you're supplied more than necessary , ignore the rest
if you're not supplied enough, the destructuring process will assign undefined to the variable that couldn't be assigned a value.
For example ```let args = [1,2]``` and suppose you were expecting three params in your destructuring process, you'll have ``` const [b, c, d] = args```. b will get the value of 1, c will get the value of 2 and d will get undefined.
As a result, checking just the types should be enough. Implementing this will also eliminate one of the similar error msgs

Reply:
all done